app/models/ingest_request.rb

```diff
@@ -8,14 +8,9 @@ class IngestRequest < ApplicationRecord
   validate :not_already_pending_validation, if: :new_record?
 
   attr_accessor :test_mode
-  after_create :send_to_fedora
 
   private
 
-  def send_to_fedora
```

Comment:
if the delayed job doesn't get created here, where does the magic happen? 

I think this might be renamed, but deleting it might prevent any / all processing?

Reply:
Ah yeah, nice find. It should be renamed, not removed.